### PR TITLE
update GitHub profiles

### PR DESCRIPTION
**Pull request by zmughal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -327,11 +331,12 @@ Hackathon Hackers' GitHub profiles
 - Anthony Lobko https://github.com/Amelius15
 - Anthony Zhang https://github.com/Uberi
 - Anton Gilgur https://github.com/agilgur5
-- Anton Relin https://github.com/relisher
 - Antonio Marino https://github.com/antonio0
+- Anton Relin https://github.com/relisher
```
Comment (arirawr) [inline README.md]: Hey - anton comes before antonio! Please fix!

File: README.md [hunk 2]
```diff
@@ -495,11 +520,14 @@ Hackathon Hackers' GitHub profiles
 - Jack Cook https://github.com/jackcook
```
Comment (arirawr) [inline README.md]: Fix alphabetical-ness plz

File: README.md [hunk 3]
```diff
@@ -513,6 +541,7 @@ Hackathon Hackers' GitHub profiles
 - Jaxon Stevens https://github.com/LemonaInc
 - Jax Reiff https://github.com/jaxuru
```
Comment (arirawr) [inline README.md]: Alphabetization.

File: README.md [hunk 4]
```diff
@@ -513,6 +541,7 @@ Hackathon Hackers' GitHub profiles
...
 - Jaydeep Singh https://github.com/Jsinghdreams
+- Jay Kamat https://github.com/jgkamat
```
Comment (arirawr) [inline README.md]: Alphabetization

File: README.md [hunk 5]
```diff
@@ -561,6 +593,7 @@ Hackathon Hackers' GitHub profiles
 - Karan Goel https://github.com/karan
 - Karl Coelho https://github.com/karlcoelho
 - Karthikeyan NG http://github.com/intrepidkarthi
```
Comment (arirawr) [inline README.md]: Duplicate

File: README.md [hunk 6]
```diff
@@ -594,12 +629,16 @@ Hackathon Hackers' GitHub profiles
 - Laurence Welch https://github.com/Lwelch45
 - Lawrence Lin Murata https://github.com/lawrencelm
 - Leo Rudberg https://github.com/LOZORD
+- Lex Alexander https://github.com/lexalexander10
 - Liam Horne https://github.com/snario
+- Liam Rahav https://github.com/LiamRahav
 - Liam Sargent https://github.com/themailman05
+- Liang Gao https://github.com/ricek
 - Liban Mohamed https://github.com/Libanthedev
 - Li Xuanji https://github.com/zodiac
```
Comment (arirawr) [inline README.md]: alpha.

**Comment by zmughal** [comment]
Good eye on spotting the dupe! :-) `uniq` didn't catch it because it was using `http` vs. `https` in the URL. :-P

Also, I resorted the list using Vim's `:sort`. Apparently GNU coreutils `sort(1)` doesn't sort like a dictionary... it skips the whitespace and uses the characters after. O_O

**Comment by bryangarza** [comment]
thanks @zmughal and @arirawr... merging 👍

**Pull request merged**